bin/pycbc_live

```diff
@@ -181,6 +181,9 @@ class LiveEventManager(object):
                                              data_readers, bank, followup_ifos)
 
             live_ifos = [ifo for ifo in ifos if 'snr_series' in fud[ifo]]
+            for ifo1, ifo2 in itertools.combinations(live_ifos, 2):
```

Comment:
I recommend using a one-liner with set comprehension here:
```python
assert len({fud[ifo]['snr_series'].delta_t for ifo in live_ifos}) == 1
```

Reply:
Yes, this willl do.
```suggestion
           assert len({fud[ifo]['snr_series'].delta_t for ifo in live_ifos}) == 1
```